tests/test_workflow.py

```diff
@@ -22,11 +61,20 @@ def test_preprocess():
         "tests/test_data/verify_output/2022-08-31_fluxes_storages.nc"
     )
     output = xr.open_dataset(output_path)
+
+    diff_figure_path = Path("tests/tmp/output_data/figures/preprocess_precip_diff.png")
+    plot_difference(
+        expected_output["precip"].sum("time"),
+        output["precip"].sum("time"),
+        diff_figure_path,
+    )
+
     numpy.testing.assert_allclose(
         expected_output["precip"].values,
         output["precip"].values,
         err_msg=(
             "Output results are different! Please verify your results. \n"
+            f"A difference plot is available under {diff_figure_path}. \n"
```

Comment:
Just a note, but further down below, on line 88, I get a failing check if there were log files present from previous test runs. Perhaps we should clean them up at the end of the tests? We could address that here or in another PR

Reply:
AFAIK I did not introduce the failing check if log files were previously present. That was already there. In the next PR I intend to improve how the log files are checked.